README.md

```diff
@@ -1 +1,103 @@
-# exitdir
+# ExitDir
+
+ExitDir is a library to signal a process should exit via the filesystem.
+
+## Demo
+
+This functionally is shown locally by using a temp directory and two processes:
+
+```shell
+export EXIT_DIR=`mktemp -d`
+go run ./cmd/busy &
+go run ./cmd/work
+```
```

Comment:
I wonder if rather than showing the demo/test code, it would make more sense to describe how this should be implemented in one's own go program briefly?

Reply:
yeah, I can do that